src/liger_kernel/chunked_loss/orpo_loss.py

```diff
@@ -98,15 +41,18 @@ def forward(
         Handles both the forward and backward pass of the final linear layer with ORPO loss.
         Inspired from LigerFusedLinearCrossEntropyFunction (https://arxiv.org/abs/2410.10989) which fuses final linear layer and CE loss.
         """
-        orpo_loss_fn = partial(
-            _compute_orpo_loss,
-            full_target=target,
+
+        return LigerFusedLinearPreferenceBase.forward(
+            ctx,
+            _input,
+            weight,
+            target,
+            bias,
```

Comment:
I wrongly assumed that forward just takes positional arguments. 
Could you also make input, target, bias keyword args?

Reply:
Done!